### ISSUE TITLE
Grid selection KB interactions vs. default interactions of widgets inside grid cells

### ISSUE THREAD

**Issue by 0ddfell0w** [w3c/aria-practices#678]
Looking for a clarification about a situation in which a widget inside a grid cell has keyboard interactions that conflict with the grid's keyboard interactions.

Suppose I have a layout grid that supports cell selection (e.g. chips component/pill list like in ). Normally, hitting Ctrl+A on an input would select all of the text. 

Code example: https://codepen.io/0ddfell0w/full/KRoRRP 
What do we think should happen when you type Ctrl+A in the input (with other cells present)? Should it select the text or select the cells?

**Comment by jnurthen** [comment]
Normally things like this work bottom-up
If the control you are in (text input) would consume ctrl+A it should continue to do so. This is the same as on any web page. If I am in a textinput then ctrl+A will select all of its text, but if I am not it will select the text on the base page.

**Comment by 0ddfell0w** [comment]
@jnurthen I agree. I should have asked explicitly: in the case where the input has no text, and thus Ctrl+A is ignored by the input, should we follow through with the grid interaction?

**Comment by jnurthen** [comment]
IMO no - just like in a textbox on the web (like this one on github) I would not expect ctrl+a to affect its parent. I'm going to 

I'll leave this open as we could perhaps clarify somewhere in section 5.9.2 what to do when a key could be consumed by different components within the page.

**Comment by mcking65** [comment]
@0ddfell0w, it is worth noting that the input in the pill list example is not inside the grid. That is intentional for multiple reasons, primarily the complications associated with putting an input inside of an edit, but also because it makes navigating to the input faster.
